src/components/report-problem.component.js

```diff
@@ -0,0 +1,117 @@
+'use strict';
+
+import React, {
+  Component,
+  View,
+  StyleSheet,
+  TouchableHighlight,
+  Text
+} from 'react-native';
+import s from './report-problem.style';
+const styles = StyleSheet.create(s);
+
+var Camera = require('react-native-camera');
+
+var RNFS = require('react-native-fs');
```

Comment:
When you say "streaming is slow" what do you mean specifically? Converting the images to base64 locally? Or do you mean once they are encoded, sending them up over the wire to Firebase is slow?